apps/heft/src/schemas/webpack.schema.json

```diff
@@ -15,6 +15,11 @@
     "webpackConfigFilePath": {
       "type": "string",
       "description": "A path to a Webpack configuration JS file. If this isn't specified, and a Webpack configuration isn't specified via another plugin, Webpack won't be run."
+    },
+
+    "webpackServeConfigFilePath": {
+      "type": "string",
+      "description": "A path to a Webpack configuration JS file, specifically to be used in --serve mode. If this and webpackConfigFilePath aren't specified, and a Webpack configuration isn't specified via another plugin, Webpack won't be run in --serve mode."
```

Comment:
If `webpackServeConfigFilePath` is not specified, would it be better to default to using `webpackConfigFilePath`? Or to report an error?

It seems like a potentially confusing experience for `heft start` to spin up but fail to run the web server. (?)

Reply:
It defaults to using the other config file. The docs say "If this **and webpackConfigFilePath** aren't specified ..."

Reply:
We could standardize the config file names:

- **webpack.config.js** - if this file is present, then Heft "just works" without the need for **webpack.json** at all (unless you need some advanced setting).
- **webpack.dev.config.js** - if this file is present, it's used by default for **webpackServeConfigFilePath**.  Otherwise, **webpackConfigFilePath** is used in `dev-server` mode